README.md

```diff
@@ -147,6 +76,50 @@ $ oc adm policy add-scc-to-user privileged -z istio-cni -n kube-system
 ```
 
 
+### Istio CNI Install Decoupled from Istio Installation
+
+The following are the steps use the CNI plugin via a separate installation process from Istio.
+
+1. clone this repo
+
+1. Install Istio control-plane
+
+1. Create Istio CNI installation manifest--either manually or via Helm:
+
+   1. (Helm Option) Construct a `helm template` or `helm install` command for your Kubernetes environment
+   
+      ```sh
```

Comment:
\`\`\`sh -> \`\`\`console, here and below.

Reply:
global replaced